docs/source/tutorials/arch_modeling/open_cell_libraries_tutorial.rst

```diff
@@ -471,10 +474,12 @@ The simulation waveforms should look similar to the following :numref:`fig_custo
 
    Simulation Waveforms with Skywater PDK Circuit Model
 
-We have now verified that the Skywater PDK Cell Library has been instantiated and bound to the OpenFPGA architecture file. If you have any problems, please reach out to us.
+We have now verified that the Skywater PDK Cell Library has been instantiated and bound to the OpenFPGA architecture file. If you have any problems, please `contact`_ us.
 
 .. _Verification: https://openfpga.readthedocs.io/en/master/tutorials/design_flow/verilog2verification/

```

Comment:
If the link is inside the openfpga documentation, please do not use URL. Instead, use reference, e.g., :ref:`from_verilog_to_verification`